packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -77,6 +83,45 @@ import {customElement} from 'lit/decorators/custom-element.js';
 import {eventOptions} from 'lit/decorators/event-options.js';
 ```
 
+## Standard decorators { #standard-decorators }
```

Comment:
I'm leaning we should move this entire `## Standard decorators` section to be all the way at the bottom, after the `## Enabling decorators`. This is introducing new `accessor` syntax that we currently do not recommend, nor use anywhere in our code samples.

I think it should serve more as an FYI, and look if you only need to kind of section, hence at the bottom.

When we mention standard decorators during the "Enabling" portion, eg. at TS migration, and usage with Babel, we can link to it.